Sources/SwiftKuery/QueryBuilder.swift

```diff
@@ -59,27 +61,32 @@ public class QueryBuilder {
         /// Last case, add new values before it.
         case namesCount
     }
-    
+
     /// An indication whether the parameters should be numbered (e.g., '$1, $2'), or just marked
     /// with the numbered parameter marker (e.g., '?').
-    public var addNumbersToParameters = true
+    public let addNumbersToParameters: Bool
     
     /// The starting index for numbered parameters.
-    public var firstParameterIndex = 1
+    public let firstParameterIndex: Int
     
     /// An indication whether ANY on subqueries is supported.
-    public var anyOnSubquerySupported = true
+    public let anyOnSubquerySupported: Bool
     
     /// An indication whether a `DELETE` query should use `USING` clause for tables in `WITH` clause.
-    public var withDeleteRequiresUsing = false
+    public let withDeleteRequiresUsing: Bool
     
     /// An indication whether an `UPDATE` query should use `FROM` clause for tables in `WITH` clause.
-    public var withUpdateRequiresFrom = false
+    public let withUpdateRequiresFrom: Bool
```

Comment:
I changed these vars to lets primarily to avoid unintentional changes and also for possible compiler optimization. I think we should also do `let substitutions: [String]` and pass it in `init` instead of `updateSubstitutions`, but that would need changes to the plugins. What do you think?